sycl/include/CL/sycl/ordered_queue.hpp

```diff
@@ -18,8 +18,11 @@
 #include <utility>
 
 #ifdef __has_cpp_attribute
```

Comment:
This `ifdef` here to check that later we can use `__has_cpp_attribute(deprecated)`. 
Shouldn't you leave `__has_cpp_attribute(deprecated)` condition?

Reply:
It seems we do not need check for attribute he at all.

Reply:
Found out why we need has_cpp_attribute - to make it compile with vc++.